cmd/statusd/main.go

```diff
@@ -41,6 +41,7 @@ var (
 	lesEnabled        = flag.Bool("les", false, "Enable LES protocol")
 	whisperEnabled    = flag.Bool("shh", false, "Enable Whisper protocol")
 	statusService     = flag.String("status", "", `Enable StatusService, possible values: "ipc", "http"`)
+	debugAPI          = flag.Bool("debug", false, `Enable debug API endpoints under "debug_" namespace`)
```

Comment:
If I understand correctly, this flag only has an effect if `-status` is also passed, correct? In that case maybe something like this would be clearer:
> Enable "debug_" namespace in StatusService

Reply:
no, this is a completely isolated flag you can enable debug api endpoints by passing `-debug=true`

Reply:
Ah you're right. Nevermind then.